.github/workflows/crawler-cicd.yaml

```
@@ -0,0 +1,40 @@
+          docker push ${{ secrets.DOCKERHUB_USERNAME }}/${{ secrets.DOCKER_IMAGE_NAME }}:latest
+
+      - name: 🚀 EC2에서 deploy.sh 실행
+        uses: appleboy/ssh-action@master
+        with:
+          host: ${{ secrets.AWS_EC2_HOST }}
+          username: ${{ secrets.AWS_EC2_USERNAME }}
+          key: ${{ secrets.AWS_EC2_SSH_KEY }}
+          script: |
+            cd ~/
+            sudo chmod +x deploy.sh
+            sudo ./deploy.sh
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)